### ISSUE TITLE
generation from array

### ISSUE THREAD

**Issue by MHenrich1990** [nschloe/pygalmesh#203]
Hi, 
for the generation of microstructure models (representative volume elements) I'm using numpy arrays. The meshing with the  generate_from_array function works fine. However, it would be great if there was an option to define feature edges which can be used to force the elemetnts on the edges of the array, to actually have a cube. otherwise the mesh looks like this:

![Pygalmesh_test](https://user-images.githubusercontent.com/45213728/202918014-1319fabc-cdd6-4834-b474-93c6deec08c1.PNG)

Basically, what I need is a cubic shape for the surface of the array mesh as shown here:

![Pygalmesh_pure_array](https://user-images.githubusercontent.com/45213728/202918593-2944be3a-ce31-43a6-9fda-e9c557706019.PNG)

Is this somehow possible? 
Thanks!

**Comment by hm7788** [comment]
I was also looking for the same feature edge solution. Has anyone figured it out yet?

**Comment by lrineau** [comment]
In CGAL_5.6-beta1, there is that feature. See [section **4.6.3 Domains from Segmented 3D Images, with 1D Features** of the Mesh_3 chapter](https://doc.cgal.org/5.6-beta1/Mesh_3/index.html#title38).

**Edit:** sorry I thought that was an issue of https://github.com/CGAL/cgal. In `pygalmesh`, I do not know if that possibility is available, or will be soon.

**Comment by hm7788** [comment]
thank you, good to know such function in CGAL. I don't see such a function in pygalmesh yet, I am not quite familiar with C++, so it  might be challenging to implement for me. Hopefully someone could wrap it in a python version someday..

**Comment by jmetancelin** [comment]
I was also looking for proper edges detection. 
PR #210 is extending the python interface to support that 1D feature detection from CGAL C++.
